src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -39,7 +39,7 @@
                     <attribute name="groups">order:create</attribute>
                 </attribute>
                 <attribute name="openapi_context">
-                    <attribute name="summary">Pickups a new cart</attribute>
+                    <attribute name="summary">Pickups a new cart. Provided locale code has to be one of available for a particular channel. Setting different local code will result in the assignment of the default locale from the channel.</attribute>
```

Comment:
```suggestion
                    <attribute name="summary">Pickups a new cart. Provided locale code has to be one of available for a particular channel. Setting different locale code will result in the assignment of the default locale from the channel.</attribute>
```

Reply:
is it proper behaviour? IMHO if we provided locale which isn't in the channel we should get an error, not default locale...